frontend/src/main.ts

```diff
@@ -16,6 +16,9 @@ import router from '@/router';
 import 'bootstrap';
 import './assets/styles/styles.scss';
 
+import 'bootstrap/dist/css/bootstrap.css'
```

Comment:
Do we need to import this bootstrap.css again? FAM already uses bootstraps style. I think it is imported in styles.scss file already.
Could you check?

Reply:
Thanks, yes, it is imported already. Removing this.